### DISCUSSION TITLE
Link generador de repos no está funcionando

### DISCUSSION THREAD

**Discussion by IsabellaKPM** [IIC2133-PUC/2024-2#5] [Tarea 0] [answered]
Hola! No sé si será un problema mío o a más personas les estará pasando, pero cuando presiono el link que se proporcionó en el pdf para crear el repo, este me envía a una pantalla de "Page Not Found".

**Comment by tomastrivino** [comment, marked as answer]
Hola Isabella,
se acaba de actualizar. Descarga la versión nueva del enunciado y cuéntanos qué tal

**Reply by IsabellaKPM** [reply]
Hola, síi, ese me funcionó, graciaas.
